src/models/sunburst.js

```diff
@@ -170,6 +172,7 @@ nv.models.sunburst = function() {
         // simple options, just get/set the necessary values
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
+        mode:       {get: function(){return mode;}, set: function(_){mode=_;}},
```

Comment:
Please add this option to the documentation by adding it to the large json object at the bottom of examples/documentation.html